BesterLive/AppDelegate.swift

```diff
@@ -14,7 +14,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
   var window: UIWindow?
 
 
-  func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) ->Bool{
+  func application(application:UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject:AnyObject]?) ->Bool{
```

Comment:
Reporter: CHECKSTYLE
Rule: ?
Severity: WARN
File: /Users/framgia/.jenkins/jobs/BesterLive/workspace/BesterLive/AppDelegate.swift L17
 

Line should be 100 characters or less: currently 121 characters

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>